### ISSUE TITLE
thank you

### ISSUE THREAD

**Issue by Andrei-Pozolotin** [mojohaus/flatten-maven-plugin#64]
thank you for the plugin idea; just FYI:

proof-of-concept simplified plugin incarnation:
https://github.com/random-maven/flatten-maven-plugin

which resolves:
https://github.com/mojohaus/flatten-maven-plugin/issues/63
https://github.com/mojohaus/flatten-maven-plugin/issues/59
https://github.com/mojohaus/flatten-maven-plugin/issues/58
https://github.com/mojohaus/flatten-maven-plugin/issues/57
etc.

**Comment by tribbloid** [comment]
Thanks a lot Andrei, can you merge #61 as well? The maintainer of this project went silent for a long time.

**Comment by Andrei-Pozolotin** [comment]
@tchemit Tony:

1) try v 1.1.x:
https://bintray.com/random-maven/maven/flatten-maven-plugin

2) like this:
https://github.com/random-maven/flatten-maven-plugin/blob/master/src/it/test-2/pom.xml#L65

3) mojo help:
https://random-maven.github.io/flatten-maven-plugin/flatten-mojo.html

**Comment by tribbloid** [comment]
@Andrei-Pozolotin Glorious! I'm merging my ticket into yours.
Thanks a lot sir!

**Comment by mattnelson** [comment]
I am interested in a few of these changes, but I don't think this plugin should be considered dead and forked.

@Andrei-Pozolotin can you with @khmarbaise  and the other maintainers of this plugin to integrate these changes and avoid the fragmentation?

**Comment by tribbloid** [comment]
@mattnelson: I agree, this plugin (mojohaus version) supports clean mojo which is not supported by @Andrei-Pozolotin 's version. It's a waste of everyone's time to integrate features into all forks.

**Comment by khmarbaise** [comment]
I appreciated your pull request...

**Comment by Andrei-Pozolotin** [comment]
@all :  

I think there is a basic design issue with `mojohaus` version: mutable vs immutable `model`

which can be addressed by something like this:

[random-maven/flatten-maven-plugin](https://github.com/random-maven/flatten-maven-plugin/blob/master/src/main/java/com/carrotgarden/maven/flatten/FlattenMojo.java#L405
)
```
// after clone():
// do not interpolate anything any more 
// only remove content or add static content
final Model model = project.getModel().clone(); 
```
but that change does not look like as a simple PR to the `mojohaus`, but rather complete plugin redesign, which I did in `random-maven` version

**Comment by hohwille** [comment]
> thank you for the plugin idea

you are welcome :)

> but that change does not look like as a simple PR to the mojohaus, but rather complete plugin redesign, which I did in random-maven version

So what is the plan then?
According to what you are saying you are not interested in improving this plugin but want to go with your own fork?
Did you see that we are getting other PRs that we merged. This will IMHO lead to two different directions and duplicated effort.
Sorry if reactions are slow here (I am involved in ~10 other OSS projects in my rare spare time). Feel free to join mojohaus and support this plugin better directly.
As @khmarbaise said we would welcome PRs from you to improve this.

**Comment by tribbloid** [comment]
sorry I changed job and worked with gradle long time ago, almost forgot it
